src/components/Task/MassActions/EditTasks/MassActionsEditTasksModal.tsx

```diff
@@ -142,19 +159,31 @@ export const MassActionsEditTasksModal: React.FC<
                     inputProps={{ 'aria-label': t('Task Name') }}
                   />
                 </Grid>
-                <Grid item xs={12} lg={6}>
+                <Grid item xs={12} sm={6}>
+                  <TaskPhaseAutocomplete
+                    options={taskPhases}
+                    value={taskPhase}
+                    onChange={(phase) => {
```

Comment:
Should we add a test that changing the phase clears the activity type and changes its options list?